packages/ckeditor5-fullscreen/docs/features/fullscreen.md

```diff
@@ -162,6 +162,66 @@ Below you will find a customized demo:
 
 {@snippet features/fullscreen-pageless}
 
+### Enabling other editor types
+
+This section covers how to provide fullscreen mode integration for other {@link getting-started/setup/editor-types editor types} than classic and decoupled. Please note that only those two are officially supported, so the code snippets below are exemplary and may not solve all technical challenges.
+
+First, you need to create a custom class extending {@link module:fullscreen/handlers/abstracteditorhandler~AbstractEditorHandler `AbstractEditorHandler`}. Besides the optional typing improvements, the most important thing is to implement its custom {@link module:fullscreen/handlers/abstracteditorhandler~AbstractEditorHandler#_defaultOnEnter `#_defaultOnEnter()`} method. It should move the editor UI elements proper for your editor type to the fullscreen container, preferably using {@link module:fullscreen/handlers/abstracteditorhandler~AbstractEditorHandler#moveToFullscreen `#moveToFullscreen()`} helper - it will assure the elements are moved back in DOM when leaving fullscreen mode.
+
+Then, in the editor's `toggleFullscreen` command, you will need to substitute the {@link module:fullscreen/fullscreencommand~FullscreenCommand#fullscreenHandler `#fullscreenHandler`} property with an instance of your custom class. It can be done either by adding a custom plugin (like shown in the snippet below), or in the `.then()` clause when the editor is created - in both cases, the code that should be executed is:
```

Comment:
I think I'd drop everything that mentions `.then()` "integration" and would just propose plugin solution for simplicity.